.flake8

```diff
@@ -0,0 +1,96 @@
+[flake8]
+per-file-ignores =
+    # imported but unused
+    __init__.py: F401, F403
+
+max-line-length = 120
+
+# Errors to ignore   - TODO: Double check
+ignore = 
+    # Line too long
+    E501,
+    # whitespace before ','
+    E203,
+    # block comment should start with ‘# ‘ 
+    E265,
+    # too many leading ‘#’ for block comment
+    E266,
+    # do not use bare 'except'  <- Consider fix and delete ignore
+    E722
+    # do not use variables named ‘l’, ‘O’, or ‘I’
+    E741,
+    # line break before binary operator
+    W503,
+    # Missing docstring in public module
+    D100,
+    # Missing docstring in public function
+    D103,
+
+# Files to exclude:
+#   
+exclude =
+    ##### examples/
+    examples
```

Comment:
Can you please list the examples instead (i.e. examples/fuse_examples/imaging/classification/stoic21/)

Reply:
Changed it, and also fixed isic & hello_world sub dirs